volatility/framework/symbols/linux/extensions/__init__.py

```diff
@@ -388,3 +388,15 @@ def get_mnt_mountpoint(self):
 
     def get_mnt_root(self):
         return self.mnt_root
+
+
+class kobject(objects.StructType):
```

Comment:
A few more documentation strings would be handy, just to describe the object and the attached method?  If this whole object is only really usable for that plugin, it might be worth adding the class override to the symbol_table (as an example to others as to how to do it).  I'm in two minds, because this will end up being a core plugin, so again, whichever you think is better.  Not a showstopper...  5:)

Reply:
What do you mean by class override to the symbol table? Is there an example of that elsewhere ?

Reply:
Sure, so here we construct a new table with a `class_types` argument, that contains the class types that are to be overridden: https://github.com/volatilityfoundation/volatility3/blob/master/volatility/framework/plugins/windows/procdump.py#L70

The idea would be that if it's not useful for other plugins but critical for this one, this one adds the override when it needs it (by constructing its own table).  Again, it's a design decision, and either will work, just wanted to make sure it'd been considered...

Reply:
The existing table comes from the kernel itself and depending on how future development goes, it may be needed by other plugins. I would prefer to keep it as-is.

Reply:
Ok, if it's gonna be reused, that makes sense.  5:)